README.rst

```diff
@@ -8,24 +8,81 @@
 OMERO.figure
 ============
 
-OMERO.web app for creating figures from images in OMERO.
+An OMERO.web app for creating figures from images in OMERO.
 
 For full details see http://figure.openmicroscopy.org
 
 
 Requirements
-------------
+============
 
-* OMERO 4.4.x or OMERO 5.x
+* OMERO 5.2.6 or newer.
 
 Installation
-------------
+============
+
+This section assumes that an OMERO.server is already installed.
+
+Install the app:
+
+::
+
+    $ pip install omero-figure
+
+Add figure custom app to your installed web apps:
+
+::
+
+    $ bin/omero config append omero.web.apps '"omero_figure"'
+
+Display a link to 'Figure' at the top of the webclient:
+
+::
+
+    $ bin/omero config append omero.web.ui.top_links '["Figure", "figure_index", {"title": "Open Figure in new tab", "target": "_blank"}]' 
+
+Now restart OMERO.web as normal.
+
+**Warning**:
+
+OMERO.figure version 2.x requires OMERO.web **5.2.6 or newer**.
+This is due to Django Framework compatibility and to a required package reorganization in OMERO.figure in version 2.0 so the application can be distributed using Python Package Manager PyPi.
```

Comment:
Benefit of mentioning Django is that 5.2 line is still compatible with version <1.8, where figure 2 is definitely not. 

it is not `pip installable` it is `installed from PyPi`, see glossary https://packaging.python.org/installing/#installing-from-pypi